internal/core/command/config.go

```diff
@@ -89,3 +91,8 @@ func (c *ConfigurationStruct) GetLogLevel() string {
 func (c *ConfigurationStruct) SetRegistryInfo(registryInfo config.RegistryInfo) {
 	c.Registry = registryInfo
 }
+
+// SetStartupInfo updates the startup information in the ConfigurationStruct
+func (c *ConfigurationStruct) SetStartupInfo(startupInfo config.StartupInfo) {
+	c.Startup = startupInfo
+}
```

Comment:
Calling out once; several instances exist.

Is this setter required?  We obtain the initial StartupInfo values from config and potentially override them from the environment.  The resulting values only have relevance during the bootstrap/startup period.  Why update them in the configuration structure (which requires the SetStartupInfo() method addition to the contract) if we're never referencing the updated values?

Reply:
I was following the pattern of RegistryInfo, but now see how the updated config is later used.

accepted